plugins/auto-sizes/includes/improve-calculate-sizes.php

```diff
@@ -288,15 +296,43 @@ function auto_sizes_filter_render_block_context( array $context, array $block ):
 	);
 
 	if ( in_array( $block['blockName'], $provider_blocks, true ) ) {
-		$alignment = $block['attrs']['align'] ?? '';
+		// Normalize default alignment values.
+		$alignment = isset( $block['attrs']['align'] ) && '' !== $block['attrs']['align'] ? $block['attrs']['align'] : 'default';
+		// Use the defined constant for constraints.
+		$constraints = AUTO_SIZES_CONSTRAINTS;
 
-		// If the container block doesn't have alignment, it's assumed to be 'default'.
-		if ( '' === $alignment ) {
-			$context['max_alignment'] = 'default';
-		} elseif ( 'wide' === $alignment ) {
-			$context['max_alignment'] = 'wide';
+		if ( 'default' === $alignment || 'wide' === $alignment ) {
```

Comment:
Why is there a check here for specific values? Shouldn't this code be able to use `AUTO_SIZES_CONSTRAINTS` without additional limitations? Can you clarify?

Reply:
Thanks for pointing it out! It was not needed, so in https://github.com/WordPress/performance/pull/1795/commits/7757e50042f001b80504c93fdbed0ed69c493f06, I have removed the alignment value checks.